manager/manifests/fluentd.yaml

```diff
@@ -168,10 +168,15 @@ metadata:
   name: fluentd
   namespace: default
 spec:
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
```

Comment:
Is there a reason you set this to 1, rather than 0 (or left the default)? I imagine that with 0, it would ensure that there is always one replica of this pod running on each node (although I am not as familiar with how this setting works with daemonsets vs deployments).

The current behavior is to take down the old pod before bringing up the new one (i.e. `maxUnavailable` is essentially 100% now). I prefer the path you've taken; my guess is that with 2 fluentd pods we might get logs duplicated in cloudwatch, which seems less bad than dropping logs if we had a period with 0 fluentd pods running.

I think this applies to the other two as well; does setting `maxUnavailable: 0` make sense to you, or was there a reason you went with `maxUnavailable: 1`?

Reply:
I looked at this too. But i think Kubernetes does not support maxUnavailable:0 (See: https://stackoverflow.com/questions/54116493/update-a-daemonset-without-downtime-like-a-deployment). maxUnavailable: 1 is the default and I included it because of visability.

Reply:
Sounds good, thanks for looking into that!